packages/fetch-engine/package.json

```diff
@@ -12,6 +12,7 @@
     "directory": "packages/fetch-engine"
   },
   "bugs": "https://github.com/prisma/prisma/issues",
+  "enginesOverride": {},
```

Comment:
Wondering if its best to have this config in a file like: 

```
engines-override.json
```

This is because I see us(well me at least) accidentally commenting changes to this.

Reply:
We could make it an env var as well? That way you could not accidentally commit it. But I think the nice part of this is that you could run this on our CI.